alembic/versions/af00a7f19025_move_commit_sha_to_project_events.py

```diff
@@ -0,0 +1,355 @@
+        unique=False,
+    )
+
+    # start data migration
+    bind = op.get_bind()
+    session = orm.Session(bind=bind)
+    rows = (
+        session.query(PipelineModel.project_event_id, KojiBuildTargetModel.commit_sha)
+        .join(
+            KojiBuildGroupModel,
+            KojiBuildTargetModel.koji_build_group_id == KojiBuildGroupModel.id,
+        )
+        .join(
+            PipelineModel, PipelineModel.koji_build_group_id == KojiBuildGroupModel.id
+        )
+        .all()
+    )
+    rows.extend(
+        session.query(PipelineModel.project_event_id, SRPMBuildModel.commit_sha)
+        .join(SRPMBuildModel, SRPMBuildModel.id == PipelineModel.srpm_build_id)
+        .all()
+    )
+    rows.extend(
+        session.query(PipelineModel.project_event_id, CoprBuildTargetModel.commit_sha)
+        .join(
+            CoprBuildGroupModel,
+            CoprBuildTargetModel.copr_build_group_id == CoprBuildGroupModel.id,
+        )
+        .join(
+            PipelineModel, PipelineModel.copr_build_group_id == CoprBuildGroupModel.id
+        )
+        .all()
+    )
+    rows.extend(
+        session.query(PipelineModel.project_event_id, TFTTestRunTargetModel.commit_sha)
+        .join(
+            TFTTestRunGroupModel,
+            TFTTestRunTargetModel.tft_test_run_group_id == TFTTestRunGroupModel.id,
+        )
+        .join(PipelineModel, PipelineModel.test_run_group_id == TFTTestRunGroupModel.id)
+        .all()
+    )
+    rows.extend(
+        session.query(
+            PipelineModel.project_event_id, VMImageBuildTargetModel.commit_sha
+        )
+        .join(
+            VMImageBuildTargetModel,
+            VMImageBuildTargetModel.id == PipelineModel.vm_image_build_id,
+        )
+        .all()
+    )
+    for row in rows:
+        bind.execute(
+            update(ProjectEventModel)
+            .where(ProjectEventModel.id == row[0])
+            .values(commit_sha=row[1])
+        )
+
+    op.drop_index("ix_copr_build_targets_commit_sha", table_name="copr_build_targets")
+    op.drop_column("copr_build_targets", "commit_sha")
+    op.drop_column("koji_build_targets", "commit_sha")
+    op.drop_column("srpm_builds", "commit_sha")
+    op.drop_column("tft_test_run_targets", "commit_sha")
+    op.drop_index(
+        "ix_vm_image_build_targets_commit_sha", table_name="vm_image_build_targets"
+    )
+    op.drop_column("vm_image_build_targets", "commit_sha")
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column(
+        "vm_image_build_targets",
+        sa.Column("commit_sha", sa.VARCHAR(), autoincrement=False, nullable=True),
+    )
+    op.create_index(
+        "ix_vm_image_build_targets_commit_sha",
+        "vm_image_build_targets",
+        ["commit_sha"],
+        unique=False,
+    )
+    op.add_column(
+        "tft_test_run_targets",
+        sa.Column("commit_sha", sa.VARCHAR(), autoincrement=False, nullable=True),
```

Comment:
```suggestion
        sa.Column("commit_sha", sa.String(), autoincrement=False, nullable=True),
```
weren't these Strings previously?